include/ros1_bridge/bridge.hpp

```diff
@@ -69,7 +69,10 @@ get_factory(
   const std::string & ros1_type_name,
   const std::string & ros2_type_name);
 
-Bridge1to2Handles
+std::unique_ptr<ServiceFactoryInterface>
+get_service_factory(const std::string &, const std::string &, const std::string &);
+
+inline Bridge1to2Handles
```

Comment:
Why should all these functions be changed to `inline`?

Reply:
thanks for the patience:

since these methods are implemented in the definition, an `inline` is a simple way to avoid `multiple definition of` linker errors.

Reply:
I would suggest to move the definition of the functions into a `.cpp` file then.

Reply:
done!